src/publisher/api_v2_views.py

```diff
@@ -289,36 +302,40 @@ def article_fragment(request, msid, fragment_id):
         ensure(fragment_id not in reserved_keys, "that key is protected")
 
         if request.method == "POST":
-            fragment_logic.add_fragment_update_article(
-                article, fragment_id, request.data
-            )
+            if request.content_type != "application/json":
```

Comment:
where did this come from? can we ensure what content-type header journal-cms is sending us?